libs/src/sdk/api/playlists/PlaylistsApi.ts

```diff
@@ -263,26 +270,78 @@ export class PlaylistsApi extends GeneratedPlaylistsApi {
       PublishPlaylistSchema
     )(requestParameters)
 
-    // Fetch playlist
-    const playlistResponse = await this.getPlaylist({
-      playlistId: requestParameters.playlistId,
-      userId: requestParameters.userId
-    })
-    const playlist = playlistResponse.data?.[0]
+    return await this.fetchAndUpdatePlaylist(
+      {
+        userId: requestParameters.userId,
+        playlistId: requestParameters.playlistId,
+        updateMetadata: (playlist) => ({
+          ...playlist,
+          isPrivate: false
+        })
+      },
+      writeOptions
+    )
+  }
 
-    if (!playlist) {
-      throw new Error(
-        `Could not fetch playlist: ${requestParameters.playlistId}`
-      )
-    }
+  /**
+   * Add a single track to the end of a playlist
+   * For more control use updatePlaylist
+   */
+  async addTrackToPlaylist(
```

Comment:
JC are we including a reorder track method as well?

Reply:
No, I think the updateTrack method is sufficient for that purpose